src/DbConnectedResourceAbstractFactory.php

```diff
@@ -112,15 +185,23 @@ protected function getCollectionFromConfig(array $config, $requestedName)
         return $collection;
     }
 
+    /**
+     * Retrieve the resource class based on the provided configuration.
+     *
+     * Defaults to ZF\Apigility\DbConnectedResource.
+     *
+     * @param array $config
+     * @param string $requestedName
+     * @return string
+     * @throws ServiceNotCreatedException if the discovered resource class
+     *     does not exist or is not a subclass of DbConnectedResource.
+     */
     protected function getResourceClassFromConfig(array $config, $requestedName)
     {
-        $defaultClass  = __NAMESPACE__ . '\DbConnectedResource';
+        $defaultClass  = DbConnectedResource::class;
         $resourceClass = isset($config['resource_class']) ? $config['resource_class'] : $defaultClass;
         if ($resourceClass !== $defaultClass
-            && (
-                !class_exists($resourceClass)
-                || !is_subclass_of($resourceClass, $defaultClass)
-            )
+            && ! (class_exists($resourceClass) || is_subclass_of($resourceClass, $defaultClass))
```

Comment:
I think this change should be reverted, because it tries evaluate first condition inside brackets, so:
`class_exists($resourceClass) || is_subclass_of($resourceClass, $defaultClass)`
and in case class does not exist it will try evaluate `is_subclass_of`. Before it was not needed - before if class does not exists there was no check for `is_subclass_of`